explorer/interpreter/value.h

```diff
@@ -341,31 +346,36 @@ class NominalClassValue : public Value {
   static constexpr llvm::StringLiteral BaseField{"base"};
 
   NominalClassValue(Nonnull<const Value*> type, Nonnull<const Value*> inits,
-                    std::optional<Nonnull<const NominalClassValue*>> base)
-      : Value(Kind::NominalClassValue),
-        type_(type),
-        inits_(inits),
-        base_(base) {}
+                    std::optional<Nonnull<const NominalClassValue*>> base,
+                    Nonnull<const NominalClassValue** const> class_value_ptr);
 
   static auto classof(const Value* value) -> bool {
     return value->kind() == Kind::NominalClassValue;
   }
 
   template <typename F>
   auto Decompose(F f) const {
-    return f(type_, inits_, base_);
+    return f(type_, inits_, base_, class_value_ptr_);
   }
 
   auto type() const -> const Value& { return *type_; }
   auto inits() const -> const Value& { return *inits_; }
   auto base() const -> std::optional<Nonnull<const NominalClassValue*>> {
     return base_;
   }
+  auto has_vtable() const -> bool { return !vtable().empty(); }
+  auto vtable() const -> const VTable&;
+  // Returns a pointer of pointer to the child-most class value. It is updated
```

Comment:
Matching my below comment... This sentence feels like it belongs more on the member. This returns the member, but when I'm reading a class to understand constructor arguments, I would look at the member before I'd look at an accessor for that member.

Reply:
Adjusted comments.